deploy/crds/operator.ibm.com_v1alpha1_helmapi_cr.yaml

```diff
@@ -10,7 +10,8 @@ spec:
       enabled: true
     image:
       pullPolicy: IfNotPresent
-      repository: quay.io/opencloudio/icp-audit-service
+      imageRegistry: quay.io/cloudio
```

Comment:
imageRegistry should be quay.io/opencloudio

Reply:
whoops left that out